src/resolver.ts

```diff
@@ -7,7 +7,10 @@ export const replaceFileExt = (
   file: string,
   extension: ValidExtensions,
 ): string => {
-  return format({ ...parse(file), base: "", ext: extension });
+  return format({ ...parse(file), base: "", ext: extension }).replace(
```

Comment:
@deckchairlabs -- I think this is needed -- maybe a better solution is to `new URL(file, 'http://localhost')`, and then remove the localhost?

Ultimately they need to be relative url-safe returns